test/upgrade/upgrade_test.go

```diff
@@ -0,0 +1,98 @@
+// +build upgrade
+
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package upgrade
+
+import (
+	"github.com/openshift-knative/serverless-operator/test/upgrade/installation"
+	"os"
+	"testing"
+
+	"go.uber.org/zap"
+	_ "knative.dev/pkg/system/testing"
+	pkgupgrade "knative.dev/pkg/test/upgrade"
+	servingupgrade "knative.dev/serving/test/upgrade"
+	eventingupgrade "knative.dev/eventing/test/upgrade"
+)
+
+func TestServerlessUpgrade(t *testing.T) {
+	c := newUpgradeConfig(t)
+	suite := pkgupgrade.Suite{
+		Tests: pkgupgrade.Tests{
+			PreUpgrade:    preUpgradeTests(),
+			PostUpgrade:   postUpgradeTests(),
+			Continual:     continualTests(),
```

Comment:
Can we just inline the functions here? i.e.

```suggestion
			PostUpgrade:   []pkgupgrade.Operation{servingupgrade.ServingPostUpgradeTests()..., eventingupgrade.PostUpgradeTest()...},
			Continual:     []pkgupgrade.BackgroundOperation{servingupgrade.ContinualTests()..., eventingupgrade.ContinualTest()...},
```